Framework/WorkflowAlgorithms/inc/MantidWorkflowAlgorithms/AlignAndFocusPowder.h

```diff
@@ -112,6 +113,7 @@ class DLLExport AlignAndFocusPowder
   int m_resampleX{0};
   std::vector<double> m_dmins;
   std::vector<double> m_dmaxs;
+  std::vector<double> m_delta;
```

Comment:
Since this has heavy conceptual overlap with `m_params`, a more distinct name would be helpful. I think `m_ragged_params` would help associate and differentiate this with the other parameter.